plugins/org.jkiss.dbeaver.ui.editors.sql/plugin.xml

```diff
@@ -4,6 +4,7 @@
 <plugin>
 
     <extension-point id="org.jkiss.dbeaver.sql.covert" name="%extension-point.org.jkiss.dbeaver.sql.covertname" schema="schema/org.jkiss.dbeaver.sql.convert.exsd"/>
+    <extension-point id="org.jkiss.dbeaver.sql.executors" name="%org.jkiss.dbeaver.sql.executors.name" schema="schema/org.jkiss.dbeaver.sql.executors.exsd"/>
```

Comment:
This extension point is missing the schema definition, please create it (you can use Eclipse for UI configurator)

Reply:
Fixed